CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # bedrock-test ChangeLog
 
+## 8.1.0 - 2022-xx-xx
```

Comment:
Why not `8.0.4` if this is just a fix?

Reply:
I can change that.  The patch I started with was complex enough to seem like more than a simple fix.  It's better now.